modules/ROOT/pages/tools/neo4j-admin/neo4j-admin-import.adoc

```diff
@@ -412,10 +412,30 @@ label:enterprise-only[]
 Incremental import is not supported for `block` format yet.
 ====
 
-When the initial data load cannot be completed in a single full import, incremental import allows the operation to be completed as a series of smaller imports.
+Incremental import allows you to incorporate large amounts of data in batches into the graph.
+You can run this operation as part of the initial data load when it cannot be completed in a single full import.
+Besides, you can update your graph by importing data incrementally, which is more performant than transactional insertion of such data.
 
-Incremental import requires the use of `--force` because it must only be run on databases in precisely the expected state and as part of an initial load.
-When an incremental import fails, it can leave the data corrupted.
+Incremental import requires the use of `--force` and can be run on an existing database only.
+
+You must stop your database, if you want to perform the incremental import within one command.
+
+If you cannot afford a full downtime of your database, split the operation into several stages:
+
+* _prepare_ stage (offline)
+* _build_ stage (offline or read-only)
+* _merge_ stagen(offline)
+
+The database must be stopped for the `prepare` and `merge` stages.
+During the `build` stage, the database can be left online but put into read-only mode.
+For a detailed example, see <<incremental-import-stages>>.
+
+Note that your database should have correct uniqueness constraints for the affected nodes, but the tool will tell you and exit, if such is missing.
```

Comment:
```suggestion
[WARNING]
====
Your database must have correct uniqueness constraints for the affected nodes.
Otherwise, the command will throw an error and exit.
====
```

Reply:
@renetapopova, I decided to proceed with your previous suggestion.😁 Otherwise, we'll get three warnings in one section.